### PR TITLE
Add an API to retrieve storage passwords

### PR DESCRIPTION
**Pull request by bmonjoie**
To write data on storage devices like NFC cards in a secure way, we need to share a password with the authenticated users of our application.

Related JIRA tickets : WC2-48

## Self proofreading checklist

- [X] Did I use eslint and black formatters
- [X] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] Did I add translations
- [X] My migrations file are included
- [X] Are there enough tests

## Changes

I have added a new model to store the passwords.
I have exposed the modification through the admin.
I made a new mobile endpoint to retrieve the passwords.

## How to test

Add a password or more to some projects.
Retrieve them through `/api/mobile/storage/passwords`

### CODE REVIEW

File: iaso/admin.py [hunk 1]
```diff
@@ -500,6 +501,19 @@ class StorageDeviceAdmin(admin.ModelAdmin):
     ]
 
 
+class StoragePasswordAdmin(admin.ModelAdmin):
+    fields = (
```
Comment (mathvdh) [inline iaso/admin.py:505]: Je ne crois pas que t'es obligé de specifier tous les fields @bmonjoie , par défaut il les prends tous ...

**Review by mathvdh — APPROVED** [review]
Except for my little comment above which is more informative than anything this is really clean .. I would say even cleaner that what the backends usually do themselves @bmonjoie !

File: iaso/admin.py (hunk 1 above)
Comment (mathvdh) [reply iaso/admin.py:505]: Sauf si tu veux changer l'ordre ou faire des groupes : https://docs.djangoproject.com/en/4.1/ref/contrib/admin/#django.contrib.admin.ModelAdmin.fields

Comment (bmonjoie) [reply iaso/admin.py:505]: OK, merci, je ne savais pas :slightly_smiling_face:

**Review by SimonDus — APPROVED** [review]
I have to admit that Benjamin is raising the bar. Everything works locally, so Martin said no more " looks good to me " but everything looks good in this case

**Pull request merged**